src/Standards/Squiz/Sniffs/Classes/ClassFileNameSniff.php

```diff
@@ -44,22 +43,27 @@ public function register()
      */
     public function process(File $phpcsFile, $stackPtr)
     {
-        $fullPath = basename($phpcsFile->getFilename());
-        $fileName = substr($fullPath, 0, strrpos($fullPath, '.'));
-        if ($fileName === '') {
-            // No filename probably means STDIN, so we can't do this check.
-            return;
+        $filename = $phpcsFile->getFilename();
+        if ($filename === 'STDIN') {
+            return $phpcsFile->numTokens;
         }
 
-        $tokens  = $phpcsFile->getTokens();
-        $decName = $phpcsFile->findNext(T_STRING, $stackPtr);
+        $fullPath  = basename($filename);
```

Comment:
Using `$filename` here and `$fileName` on line 52 feels like asking for trouble. Maybe `$absoluteFileName` here?

Reply:
@braindawg Valid point. Thanks for catching that. I originally stuck to the variables names which were already being used in the sniff to make the change as small as possible, but you are 100% correct, this is error prone and the names are a bit confusing anyway.

I've now added a new commit to use more meaningful variable names.

I've also rebased as some CI checks have changed, so a rebase is needed to get a passing build, but the original commits have not been changed.